kms/api/Kms.Samples/CreateKeyAsymmetricDecrypt.cs

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START kms_create_key_asymmetric_decrypt]
+using Google.Cloud.Kms.V1;
+
+public class CreateKeyAsymmetricDecryptSample
+{
+    public CryptoKey CreateKeyAsymmetricDecrypt(
+      string projectId = "my-project", string locationId = "us-east1", string keyRingId = "my-key-ring",
```

Comment:
The linter doesn't seem to complain, and neither does the linter on CI. I'm happy to change it, but it seems like it won't be enforced since neither linter actually cares.